src/components/AnnotationPopover/AnnotationPopover.js

```diff
@@ -60,18 +63,28 @@ class AnnotationPopover extends React.PureComponent<Props> {
             onCreate,
             onCommentClick,
             language,
-            messages: intlMessages
+            messages: intlMessages,
+            headerHeight
         } = this.props;
         const hasComments = comments.length > 0;
 
         return (
             <Internationalize language={language} messages={intlMessages}>
                 <div
                     className={classNames('ba-popover', {
-                        'ba-inline': isMobile || (!isPending && !hasComments)
+                        'ba-inline': !hasComments && (isMobile || !isPending)
                     })}
                 >
-                    <span className='ba-popover-caret' />
+                    {isMobile ? (
+                        <span className='ba-mobile-header' style={{ height: headerHeight }}>
```

Comment:
Would it make more sense to set the height of the button to be something in css like `100%`, instead of `headerHeight` being passed in?

Reply:
Set it to match the default preview header height

Reply:
It was configurable in the case of preview headers that vary in size.